core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -249,7 +250,8 @@ public void registerCleansUpOrphanedBlocks() throws Exception {
     // Check that the worker heartbeat tells the worker to remove the blocks.
     alluxio.grpc.Command heartBeat = mBlockMaster.workerHeartbeat(workerId, null,
         memUsage, NO_BLOCKS, NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE, mMetrics);
-    assertEquals(orphanedBlocks, heartBeat.getDataList());
+    assertEquals(orphanedBlocks,
```

Comment:
HashSet should not guarantee a fixed ordering. From java docs

(http://docs.oracle.com/javase/6/docs/api/java/util/HashSet.html):

This class implements the Set interface, backed by a hash table (actually a HashMap instance). It makes no guarantees as to the iteration order of the set; in particular, it does not guarantee that the order will remain constant over time. This class permits the null element.